src/default.pre.js

```diff
@@ -34,6 +37,32 @@ function collectMetadata(ctx) {
     });
 };
 
+/**
+ * Collects the nav and append it to the resource
+ * @param {RequestContext} ctx Context
+ */
+function collectNav(ctx) {
+    const params = {
+        org: ctx.strandConfig.urls.content.owner,
+        repo: ctx.strandConfig.urls.content.name,
+        tree: ctx.strandConfig.urls.content.ref,
+        path: 'SUMMARY.md'
+    };
+
+    return md2json.main(params).then(info => {
```

Comment:
@kptdobe I think this is the wrong way to do it – every template script should render only one single resource and never try to go back to the repository. 
You can handle this much better though an edge-side include (take a look at the nav.md example in `template2lamdba`, with better overall performance.

Reply:
I've filed adobe/project-helix#171 and #4 for this

Reply:
Yes, I agree. Step by step... for now, that's the only way to do it.